### PR TITLE
[RFR] test_provisioning_dialog.py Provision fix

### PR DESCRIPTION
**Pull request by slaykovsky**
Fixing provisioner closure because it is currently fails on form filling.

Signed-off-by: Aleksei Slaikovskii <aslaikov@redhat.com>

{{pytest: -v --long-running cfme/tests/infrastructure/test_provisioning_dialog.py}}

### CODE REVIEW

**Comment by lkhomenk** [comment]
this PR can be as hot-fix for this module. but it also need to migrate to WT and new provisioning form usage

**Comment by slaykovsky** [comment]
Making it RFR because something is wrong on CI...

File: cfme/infrastructure/virtual_machines.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
 from widgetastic_patternfly import (
     Button, BootstrapSelect, BootstrapSwitch, Dropdown, Input as WInput, Tab)
 from widgetastic_manageiq import (
-    TimelinesView, Accordion, ManageIQTree, SummaryTable, ConditionalSwitchableView)
+    NonJSPaginationPane, TimelinesView, Accordion,
+    ManageIQTree, SummaryTable, ConditionalSwitchableView)
```
Comment (mkoura) [inline cfme/infrastructure/virtual_machines.py]: Imports should be sorted alphabetically.

File: cfme/tests/infrastructure/test_provisioning_dialog.py [hunk 2]
```diff
@@ -244,8 +252,10 @@ def test_tag(provisioner, prov_data, provider, vm_name):
     vm = provisioner(template_name, prov_data)
 
     tags = vm.get_tags()
-    assert any(tag.category.display_name == "Service Level" and tag.display_name == "Gold"
-               for tag in tags), "Service Level: Gold not in tags ({})".format(str(tags))
+    assert any(
+        tag.category.display_name == "Service Level" and tag.display_name == "Gold"
+        for tag in tags
+    ), "Service Level: Gold not in tags ({})".format(str(tags))
```
Comment (mkoura) [inline cfme/tests/infrastructure/test_provisioning_dialog.py]: OPTIONAL: explicit ``str`` is not needed when using ``format``.

Comment (slaykovsky) [reply cfme/tests/infrastructure/test_provisioning_dialog.py]: Thanks, done.

File: cfme/infrastructure/virtual_machines.py (hunk 1 above)
Comment (slaykovsky) [reply cfme/infrastructure/virtual_machines.py]: Done.

**Comment by mkoura** [comment]
Waiting for PRT results before marking it as 1LP.

**Pull request merged**